back/app/controllers/web_api/v1/admin_publications_controller.rb

```diff
@@ -18,9 +18,9 @@ def index
     )
       .joins('LEFT OUTER JOIN admin_publications AS parents ON parents.id = admin_publications.parent_id')
       .order('root_ordering, admin_publications.ordering')
-      .includes(:publication, :children)
 
     @publications = paginate publications
+    @publications = @publications.includes(:publication, :children)
```

Comment:
Is it worth adding a comment after (or above) each of these cases where you moved these `includes` to be after pagination?
```
@publications = @publications.includes(:publication, :children) # Using `includes` after pagination avoids rails preloading items it does not need, thus improving performance.
```
You did this in the `IdeasController`. I'm just worried people will undo this/these, without understanding the impact.